pkg/runtime/wfengine/activity.go

```diff
@@ -235,7 +235,7 @@ func (a *activityActor) loadActivityState(ctx context.Context, actorID string, g
 	}
 
 	// Loading from the state store is only expected in process failure recovery scenarios.
-	wfLogger.Infof("%s: loading activity state", actorID)
+	wfLogger.Debugf("%s: loading activity state", actorID)
```

Comment:
This one we need to investigate further. I was not expecting we'd ever see this log in the normal execution path (as hopefully can be understood from the comment above). I've taken a note to follow up on this to see if there is a performance issue that needs to be made here.

Reply:
It may be a leftover log that was used to debug this while programming. If you don't think it matters, feel free to remove them.

Our logs are not part of any API contract anyways, with very few exceptions (like API logs for the HTTP/gRPC servers)